js/tab.js

```diff
@@ -1,13 +1,53 @@
-function openTab(evt, cityName){
-    var i, tabcontent, tablinks;
+//The manageHash function deals with the content changes on hitting a url with a particular tab id
+function manageHash(defaultID){
```

Comment:
I still don't know what the "Hash" that is managed is: reading the function does not make it clear to me. Could you explain?

Reply:
In order to toggle the contents of the tabs, we need to know which tab is clicked or active. But when the user directly enters the URL in the browser manually, we need to recover the hash from URL entered (P.S. The hash is also the id of the active tab). 
The `manageHash` function deals with the tab content changes when the user enters URL manually in the browser with a particular hash/(or, tab id)